src/tpl/gotham/gotham.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Gotham Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package gotham
+
+import (
+	"encoding/base64"
+	"html/template"
+
+	qrcode "github.com/skip2/go-qrcode"
+)
+
+func New() *Namespace {
+	return &Namespace{}
+}
+
+type Namespace struct{}
+
+func (ns *Namespace) QRCoder(url interface{}) (template.HTML, error) {
+
+	var png []byte
```

Comment:
codecov is saying that several lines in this file are not covered by tests. should we ignore?

Reply:
No you and Codecov are correct. There is a file I left out because it looks like the init file has the tests. But apparently in Hugo those tests are duplicated. So I need to add it. I'll get to this last piece tomorrow/Thursday.

Reply:
Completed.